edge/pkg/devicetwin/dtmanager/communicate.go

```diff
@@ -152,13 +152,10 @@ func (cw CommWorker) checkConfirm(context *dtcontext.DTContext, msg interface{})
 	context.ConfirmMap.Range(func(key interface{}, value interface{}) bool {
 		dtmsg, ok := value.(*dttype.DTMessage)
 		klog.V(2).Info("has msg")
-		if !ok {
-
-		} else {
```

Comment:
If not ok, do we need to print err log and return false? :)

Reply:
Maybe `warining` log?

Reply:
The log can be printed, but there should be no need to return false. Because the confirmMap may still have some messages that are normally waiting to be processed.

Reply:
/approve